mitxpro/settings.py

```diff
@@ -901,6 +900,9 @@
     default=200,
     description="The maximum number of cache entries for Wagtail images",
 )
+
+WAGTAILADMIN_BASE_URL = SITE_BASE_URL
```

Comment:
It was added by Nathan and I just kept it. Wagtail has renamed its setting from `BASE_URL` to `WAGTAILADMIN_BASE_URL` and by reading the docs it looks like it should be pointing to `/cms` but I am not 100% clear on this one. We can remove this as well but its a good one to keep to avoid fallback options.

https://docs.wagtail.org/en/latest/reference/settings.html#wagtailadmin-base-url

Let me know what you think of this.